nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/UserGroupTest.java

```diff
@@ -378,8 +378,8 @@ public void itDoesntWritePassword() throws Exception {
         // When I call JSON for user1
         JsonNode node = getResponseAsJson(RequestType.GET, "/user/user1");
 
-        // Then it doesn't contain the password
-        assertEquals("", node.get("properties").get("password").getValueAsText());
+        // Then it doesn't contain the password at all
+        assertNull("", node.get("properties").get("password"));
```

Comment:
No need for an empty message here
`assertNull(node.get("properties").get("password"));`